mix.exs

```diff
@@ -22,13 +23,11 @@ defmodule EctoVista.MixProject do
 
       # Docs
       name: "EctoVista",
-      source_url: "https://github.com/flowerett/ecto_vista",
```

Comment:
Any reason for removing the `source_url`?
The official documentation [suggests to have it both](https://hexdocs.pm/ex_doc/Mix.Tasks.Docs.html#module-configuration).
Or maybe if they are identical, source_url can be omitted?

Reply:
Good to put within the `docs` key instead of the root level as we want to group all documentation related options together. If we look the how `ex_doc` process the module config, additional and selected options from root level are merged in to default `docs` options.

See https://github.com/elixir-lang/ex_doc/blob/master/lib/mix/tasks/docs.ex#L333.

```
    options =
      config
      |> get_docs_opts() <===========
      |> Keyword.merge(cli_opts)
      # accepted at root level config
      |> normalize_source_url(config) <==========
      # accepted at root level config
      |> normalize_homepage_url(config) <============
      |> normalize_source_beam(config)
      |> normalize_apps(config)
      |> normalize_main()
      |> normalize_deps()

......

  defp normalize_source_url(options, config) do
    if source_url = config[:source_url] do
      Keyword.put(options, :source_url, source_url)
    else
      options
    end
  end
```

Reply:
Then I would rather remove it from the docs section.

Reply:
> Then I would rather remove it from the docs section.

Done. Just to share what I've discovered, why `:source_url` can be added to root level config, see https://github.com/elixir-lang/ex_doc/pull/718.